pkg/cmd/server/start/node_args.go

```diff
@@ -152,12 +152,6 @@ func ValidateRuntime(config *configapi.NodeConfig, components *utilflags.Compone
 
 	switch strings.ToLower(config.NetworkConfig.NetworkPluginName) {
 	case ovs.MultiTenantPluginName:
-		if actual.Has(ComponentKubelet) && !actual.Has(ComponentPlugins) {
-			return fmt.Errorf("the multi-tenant SDN plugin must be run in the same process as the kubelet")
-		}
-		if actual.Has(ComponentProxy) && !actual.Has(ComponentPlugins) {
-			return fmt.Errorf("the multi-tenant SDN plugin requires the proxy and plugins components be enabled in the same process")
-		}
 	case ovs.SingleTenantPluginName:
 		if actual.Has(ComponentKubelet) && !actual.Has(ComponentPlugins) {
 			return fmt.Errorf("the SDN plugin must be run in the same process as the kubelet")
```

Comment:
You're leaving the "sdn in same process as kubelet" check for single-tenant, but not for multi-tenant